tck/src/test/java/org/reactivestreams/tck/PublisherVerificationTest.java

```diff
@@ -585,6 +586,38 @@ public void required_spec317_mustSupportACumulativePendingElementCountUpToLongMa
     }, "Async error during test execution: Illegally signalling onError too soon!");
   }
 
+  @Test
+  public void required_spec317_mustNotSignalOnErrorWhenPendingAboveLongMaxValue_forSynchronousPublisher() throws Throwable {
+    final AtomicInteger sent = new AtomicInteger();
+
+    customPublisherVerification(new Publisher<Integer>() {
+      @Override
+      public void subscribe(final Subscriber<? super Integer> downstream) {
+        downstream.onSubscribe(new Subscription() {
+          boolean started;
+          boolean cancelled;
+
+          @Override
+          public void request(long n) {
+            if (!started) {
+              started = true;
+              while (!cancelled) {
+                downstream.onNext(sent.getAndIncrement());
+              }
+            }
+          }
+
+          @Override
+          public void cancel() {
+            cancelled = true;
+          }
+        });
+      }
+    }).required_spec317_mustNotSignalOnErrorWhenPendingAboveLongMaxValue();
+
+    // 11 due to the implementation of this particular TCK test (see impl)
+    Assert.assertEquals(sent.get(), 11);
```

Comment:
If it is _wrong_ I'd want to see it removed, but from a cursory inspection I don't see that it could be problematic, but if I am wrong, please do tell me!